extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -637,7 +638,8 @@ private static HttpServerOptions createSslOptions(HttpBuildTimeConfig buildTimeC
                     trustStorePassword.get(),
                     sslConfig.certificate.trustStoreFileType,
                     sslConfig.certificate.trustStoreProvider,
-                    sslConfig.certificate.trustStoreCertAlias);
+                    sslConfig.certificate.trustStoreCertAlias,
+                    Optional.empty());
```

Comment:
Although vert.x allows setting a password for any entry, I don't think it makes any sense for public keys in trust stores.

Reply:
Agreed.